src/black/__init__.py

```diff
@@ -318,6 +318,12 @@ def validate_regex(
         "editors that rely on using stdin."
     ),
 )
+@click.option(
+    "-W",
+    "--workers",
+    type=int,
+    help="Number of parallel workers [default: os.cpu_count()]",
```

Comment:
This could be confusing from one machine to another and on our documentation. Probably not that big of a deal though, and it *is* convenient to see the value right away 🤔